schematic/utils/io_utils.py

```diff
@@ -1,8 +1,11 @@
 import os
 import json
 import urllib.request
+import pkg_resources
+from os import path, pathsep
+from errno import ENOENT
```

Comment:
Do we need to load these packages here? Don't they just need do be loaded in the loader.py submodule?

Reply:
Yup, you're exactly right. I left them there when I had the Loader() being called in every module. I've removed the imports now.